docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -18,85 +18,43 @@ We'll cover the following topics:
 
 ## Installation
 
-The controller package provides a service unit and, if bootstrapping is enabled, will generate a configuration based on the bootstrapping answer file. The `openziti` package provides the `ziti` CLI and is installed automatically as a dependency. Run the install script for RPM and Debian distributions or manually configure the package repo and install package `openziti-controller`.
+The controller package provides a systemd service unit and bootstrapping script.
 
-Download and run the install script.
+Run the install script.
 
 ```text
-wget https://get.openziti.io/install.bash
+curl -sS https://get.openziti.io/install.bash | sudo bash -s openziti-controller
 ```
 
-Install interactively so that you will have an opportunity to answer questions about generating a configuration.
-
-```text
-sudo bash ./install.bash openziti-controller
-```
+Or, manually configure the package repo and install `openziti-controller`.

```

Comment:
somehow, visually, this is not as clear as it could be. did you try using tabs here? it might just be our current styling, it might be presentation, but right now it doesn't feel like an "or" .  Maybe three tabs with the script install being the default? 
| script install | Debian Family | RedHat Family | 
maybe?